src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1104,6 +1104,9 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                         if ($_.Exception -and $_.Exception.WasThrownFromThrowStatement) {
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Tests for $ErrorView.ConciseView tests.Activity shows up correctly for scriptblocks
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/Formatting/ErrorView.Tests.ps1: line 54
54:             write-verbose -verbose $e[2].Replace("`e","ESC")

```